joomla/joomla-cms
# [com_config] Error?/Success? messages on save

#### Summary of Changes

In com_config when we get an error on save (ex: bad session token or no permission) we get a green (aka success) message.

This is a bad UX problem:
- Green messages are for success
- Blue messages are for information
- Yellow messages are for warnings
- Red messages are for errors or very importing stuff

This PR corrects that.
#### Testing Instructions

Code review, or, for instance:
1.  (browser Tab 1) Go to global config form, don't save
2.  (browser Tab 2) Open a new browser tab, go to backend, logout and login again
3. (browser Tab 1) Press "Save" or "Save and Close" in the global config to save.
4. You'll notice you get and invalid token message and the message is green and from a user perspective it's associated with action success.
 ![image](https://cloud.githubusercontent.com/assets/9630530/15186255/34de2070-1794-11e6-9482-ffc8377ee40c.png)
5. Apply patch, repeat process 1 to 3. Now the message is red.
 ![image](https://cloud.githubusercontent.com/assets/9630530/15186223/1813023a-1794-11e6-9c3d-9491e1c71ade.png)
